.github/workflows/extrinsic-ordering-check-from-bin.yml

```diff
@@ -1,29 +1,18 @@
 name: Release - Extrinsic Ordering Check
 
 on:
+  workflow_dispatch:
   workflow_run:
-    workflows: ["Create release draft", "Issue stale bot"]
+    workflows: ["Create release draft"]
     branches:
```

Comment:
This won't work. The workflow "Create release draft" will (almost) always run on release branch, but probably never on `dev` branch.

I think we can simply delete the branches directive.

Reply:
Compare `rococo`, `litmus` and  `litentry` on production endpoint with the parachain to be released endpoint.